eduvpn/i18n.py

```diff
@@ -9,9 +13,38 @@
 country_mapping = None
 
 
+def init(lets_connect: bool, prefix: str):
+    """
+    Init locale and gettext, returns text domain
+    """
+    domain = 'LetConnect' if lets_connect else 'eduVPN'
+    directory = os.path.join(prefix, 'share/locale')
+
+    locale.setlocale(locale.LC_ALL, '')
+    locale.bindtextdomain(domain, directory)  # type: ignore
+    locale.textdomain(domain)  # type: ignore
+    install(domain, names=('gettext', 'ngettext'), localedir=directory)
```

Comment:
This installs `gettext` and `ngettext` in the  builtins namespace.

And this seems to be the "only one way, albeit the most convenient way, to make the _() [and `gettext` and `ngettext`] function available to your application".
https://docs.python.org/3/library/gettext.html#gettext.NullTranslations.install

Reply:
The docs do **not** say "_the_ only one", but that it "is only one way".
The next line continues to say one "should never install _()".

Did you read my comment about using [`translation`](https://docs.python.org/3/library/gettext.html#gettext.translation) in the [other thread](https://github.com/eduvpn/python-eduvpn-client/pull/341#discussion_r601430437)?
Let me know if I help you :)

Reply:
I think you guessed it: I'm not sure what the goal of your proposed change is. :)

I'll try to summarize the three use cases of `gettext`, classify them as I understand them, and then make a résumé.

**Case 1**
A whole application is translated. This is then the following case:
https://docs.python.org/3/library/gettext.html#localizing-your-application

**Case 2**
If only one module of the application is translated, this is the correct approach:
https://docs.python.org/3/library/gettext.html#localizing-your-module

**Case 3**
If the application uses multiple languages at the same time, meaning one dialog would be in English, another in German, then the correct approach is:
https://docs.python.org/3/library/gettext.html#changing-languages-on-the-fly
Which actually corresponds to case 2.

**Résumé**
Using case 1 (`install(...)`) seems to be the right approach here. Of course case 2 would also work, but the implementation would require to pass a "*Translations instance" or provide it globally. This would be unnecessarily complicated.

Reply:
Thanks for the detailed response :)

My goal is to avoid globally installing  `_()`, `gettext()`, and friends.
However, I see that the function I was referring to is not `translations()` but [`bindtextdomain()`](https://docs.python.org/3/library/gettext.html#gettext.bindtextdomain).
See the first example [in the `gettext` module docs](https://docs.python.org/3/library/gettext.html).
Sorry for any confusion, I hope it's clear now.